VsCredentialProvider/Sample.VisualStudio.CredentialProvider/SampleCredentialProvider.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.VisualStudio;
+
+namespace Sample.VisualStudio.CredentialProvider
+{
+
+    [Export(typeof(IVsCredentialProvider))]
+    public sealed class SampleCredentialProvider
+        : IVsCredentialProvider
+    {
+        /// <summary>
+        /// Get credentials for the supplied package source Uri.
+        /// </summary>
+        /// <param name="uri">The NuGet package source Uri for which credentials are being requested. Implementors are
```

Comment:
It's actually already mentioned in the `<returns>` tag, so I'll just drop that sentence completely.